dockerfiles/Dockerfile.dev

```diff
@@ -0,0 +1,13 @@
+# syntax=docker/dockerfile:1
+ARG VOLTO_VERSION
+FROM plone/frontend-dev:${VOLTO_VERSION}
+
+ARG ADDON_NAME
+ARG ADDON_PATH
+
+COPY --chown=node:node package.json /app/src/addons/${ADDON_PATH}/
+
+RUN <<EOT
+    /setupAddon
+    yarn install --network-timeout 1000000
+EOT
```

Comment:
@sneridagh @ericof Dockerfile.dev is identical to Dockerfile.acceptance. What's the reasoning for 2 different Dockerfiles?